apps/playground-web/src/app/insight/[blueprint_slug]/blueprint-playground.client.tsx

```
@@ -450,10 +451,10 @@ function ParameterSection(props: {
 
           const hasError = !!props.form.formState.errors[param.name];
 
-          const placeholder = url.includes(`{${param.name}}`)
-            ? `{${param.name}}`
-            : url.includes(`:${param.name}`)
-              ? `:${param.name}`
+          const placeholder =
```

Biome (1.9.4)

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.
Unsafe fix: If this is intentional, prepend placeholder with an underscore.


(lint/correctness/noUnusedVariables)